src/lib/assets/styles/button.css

```diff
@@ -101,6 +101,22 @@ button.delete {
 button.delete:hover, button.delete:focus {
   background-color: var(--red-emphasis);
 }
+/* pagination controls */
+.pager {
+  display: inline-block;
+}
+.pager .btn-link {
```

Comment:
This is also nitpicky along with my white background opinion... This doesn't seem like a "btn-link" style anymore. A button link should be styled to look like a plain link (even though we are overriding that over in metrics with the menus -different story, that's not in a global style sheet). This is seeming more like a "ghost" button and maybe calls for a new class name.

`btn-link` is borrowed from bootstrap https://getbootstrap.com/docs/4.0/components/buttons/
This is an example of some ghost buttons https://codepen.io/mithicher/pen/acyfq

Reply:
how about using an explicit `pager-link` instead?

Reply:
Happy we settled on `button.pager`. Thanks for entertaining my attempt at some care and feeding of styleguide best practices.

Reply:
we settled on `button.pager`